targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52810/device/system_nrf52810.c

```diff
@@ -0,0 +1,394 @@
+/*
+
+Copyright (c) 2009-2017 ARM Limited. All rights reserved.
+
+    SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the License); you may
+not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an AS IS BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+NOTICE: This file has been modified by Nordic Semiconductor ASA.
+
+*/
+
+/* NOTE: Template files (including this one) are application specific and therefore expected to
+   be copied into the application project folder prior to its use! */
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "nrf.h"
+#include "system_nrf52810.h"
+
+/*lint ++flb "Enter library region" */
+
+#define __SYSTEM_CLOCK_64M      (64000000UL)
+
+static bool errata_31(void);
+static bool errata_36(void);
+static bool errata_66(void);
+static bool errata_103(void);
+static bool errata_136(void);
+
+/* Helper functions for Errata workarounds in nRF52832 */
+#if defined (DEVELOP_IN_NRF52832)
```

Comment:
Are these workarounds still needed for this part?

Reply:
Nordic don't build a dev kit for the 810, officially they say to use the normal 832 board but compile your 810 code with this `DEVELOP_IN_NRF52832` define set which allows 810 code to run on 832

I've linked this define through to a mbed config to expose it as such